### PR TITLE
Restructure docs

### PR DESCRIPTION
**Pull request by necoline**
this is a pr to restructure the docs in a better way

### CODE REVIEW

File: packages/projects-docs/pages/learn/_meta.json [hunk 1]
```diff
@@ -1,11 +1,27 @@
...
+  "index": "Introduction",
+  "getting-started": "Guides",
+  "-- cloud-development": {
+    "type": "separator",
+    "title": "Cloud Development"
+  },
+  "repositories": "",
+  "sandboxes": "",
+  "editors": "",
+  "boxy": "",
```
Comment (filipelima18) [inline packages/projects-docs/pages/learn/_meta.json:11]: This should be uppercase.

File: packages/projects-docs/pages/learn/_meta.json [hunk 2]
```diff
@@ -1,11 +1,27 @@
...
+  "environment": "",
+  "integrations": "",
+  "-- browser-sandboxes": {
+    "type": "separator",
+    "title": "Browser Sandboxes"
+  },
+  "browser-vs-cloud-sbx": "Why Browser Sandboxes",
+  "browser-sandboxes": "Features",
```
Comment (filipelima18) [inline packages/projects-docs/pages/learn/_meta.json:19]: This sounds vague. We should add something more descriptive.

**Review by filipelima18 — CHANGES_REQUESTED** [review]
- Change "packages/projects-docs/pages/learn/_meta.json" to uppercase.
- Add a more descriptive explanation at line 19 in "packages/projects-docs/pages/learn/_meta.json".

File: packages/projects-docs/pages/learn/_meta.json (hunk 2 above)
Comment (necoline) [reply packages/projects-docs/pages/learn/_meta.json:19]: good point, I'll fix this

**Pull request closed without merging**